terraform/staging/main.tf

```diff
@@ -133,4 +133,7 @@ module "staging_cluster" {
   provision_opensearch_cluster = false
 
   ses_email_domain = "staging.spack.io"
+
+  public_binary_mirrors       = false
```

Comment:
Why is this configurable? And why is it false in staging?

Reply:
I figured there was no reason to allow public access as it's a staging bucket. If it's public, i think people could point their spack application at it and use it as a binary mirror. 
Maybe we don't care about that though, I don't have any strong opinions on it

Reply:
I would vote for keeping prod/staging parity here since I think (hope?) we will be the people who point our spack installs at it. I think staging ought to be an identical-as-much-as-possible environment that just isn't very useful for non-developers.

Reply:
Sounds good to me. I changed it